doc/design/parallel-node-config.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Parallel SR-IOV configuration
+authors:
+  - e0ne
+reviewers:
+  - adrianchiris
+  - SchSeba
+creation-date: 18-07-2023
+last-updated: 18-07-2023
+---
+
+# Parallel SR-IOV configuration
+
+## Summary
+SR-IOV Network Operator configures NICs one by one. That means we’ll need to wait hours or even days to configure all
+NICs on large cluster deployments. With multi-NICs deployments Operator configures NICs one by one on each node which
+leads to a lot of unnecessary node drain calls during SR-IOV configuration.
+
+## Motivation
+Allow SR-IOV Network Operator to configure more than one node at the moment.
+
+### Use Cases
+
+### Goals
```

Comment:
one of the goals IMO should be to move draining logic from config daemon to a centralized place.